### PR TITLE
Issue #996 Create a non clickable fax numbers component

### PR DESCRIPTION
**Pull request by ian-lin-PG**

### CODE REVIEW

File: src/components/phone_numbers/phone_numbers_component.tsx [hunk 1]
```diff
@@ -48,11 +48,18 @@ const buildPhoneNumber = R.curry((props: Props, phoneNumber: PhoneNumber, index:
...
+                    leftContent={renderSinglePhoneNumber(phoneNumber)}
+                    rightContent={<EmptyComponent />}
+                    onPress={undefined} />
+                :
+                <CardButtonComponent
+                    leftContent={renderSinglePhoneNumber(phoneNumber)}
+                    rightContent={phoneIcon}
+                    onPress={callPhoneNumber}
+                />
+            }
```
Comment (rasmus-storjohann-PG) [inline src/components/phone_numbers/phone_numbers_component.tsx:62]: I'm not a fan of these long ternary expressions. Can you make a separate component that all it does is to decide which kind of card button component you get. It should still be called with the `<Foo/>` syntax, not `{foo()}`. We can talk about it if that doesn't make sense.

File: src/components/card_button_component.tsx [hunk 2]
```diff
@@ -10,14 +10,23 @@ export interface CardButtonProps {
...
-                    {props.leftContent}
-                </View>
-                <View style={{ flex: .5, alignItems: 'flex-end', paddingRight: 10 }}>
-                    {props.rightContent}
-                </View>
-            </View>
+            <NonClickableCardButtonComponent
+                leftContent={props.leftContent}
+                rightContent={props.rightContent}
+                onPress={undefined}/>
```
Comment (rasmus-storjohann-PG) [inline src/components/card_button_component.tsx:16]: I figured it was just the fax numbers that should not be clickable, here it looks like this applies more widely...

File: src/components/phone_numbers/phone_numbers_component.tsx (hunk 1 above)
Comment (ian-lin-PG) [reply src/components/phone_numbers/phone_numbers_component.tsx]: Made a `PhoneNumberComponent` with `PhoneNumberProps` that switches to fax or card button components.

File: src/components/card_button_component.tsx (hunk 2 above)
Comment (ian-lin-PG) [reply src/components/card_button_component.tsx]: I opted to backtrack on this change cause it's premature optmization. If more fields becomes unclickable we know we change change here.

**Pull request merged**